browse-scaler/src/__tests__/cmr.test.js

```diff
@@ -1,97 +1,158 @@
-const { getBrowseImageFromConcept,
-        getCollectionLevelBrowseImage,
-        getGranuleLevelBrowseImage,
-        getEchoToken } = require('../cmr');
-const config = require ('../config');
+import nock from 'nock';
+// import fs from 'fs';
+import {
+  getBrowseImageFromConcept,
+  getCollectionLevelBrowseImage,
+  getGranuleLevelBrowseImage
+} from '../cmr.js';
+import { CMR_ROOT_URL } from '../config.js';
 
-const nock = require ('nock');
-const AWS = require('aws-sdk-mock');
+import collectionWithBrowse from './__mocks__/C179003030-ORNL_DAAC.js';
+import collectionWithoutBrowse from './__mocks__/C1214587974-SCIOPS.js';
+import granuleWithBrowse from './__mocks__/C1711961296-LPCLOUD_granules.js';
+import granuleWithBrowseMultipleImages from './__mocks__/G1200460416-ESA.js';
+import granuleWithoutBrowse from './__mocks__/C179003030-ORNL_DAAC_granules.js';
+import invalidCollectionIdResponse from './__mocks__/C000000000-MISSING_NO.js';
+import invalidGranuleIdResponse from './__mocks__/G000000000-MISSING_NO.js';
+// import { feed as _feed } from './C1214587974-SCIOPS.json';
 
-const collectionWithBrowse = require('./C179003030-ORNL_DAAC.json');
-const collectionWithoutBrowse = require('./C1214587974-SCIOPS.json');
+// const { collectionWithBrowse } = JSON.parse(fs.readFileSync(`${__dirname}/C179003030-ORNL_DAAC.json`));
 
-const granuleWithBrowse = require('./C1711961296-LPCLOUD_granules.json');
-const granuleWithoutBrowse = require('./C179003030-ORNL_DAAC_granules.json');
+// const { collectionWithoutBrowse } = require('./C1214587974-SCIOPS.json');
 
-const invalidCollectionIdResponse = require('./C000000000-MISSING_NO.json');
-const invalidGranuleIdResponse = require('./G000000000-MISSING_NO.json');
+// const { collectionWithoutBrowse } = JSON.parse(fs.readFileSync(`${__dirname}/C1214587974-SCIOPS.json`));
 
-describe('Metadata wrangling', () => {
-    test('Get image url from collection with browse url', async () => {
-        const imageUrl = await getBrowseImageFromConcept(collectionWithBrowse.feed.entry[0]);
-        expect(imageUrl).toBe(
-            'https://daac.ornl.gov/graphics/browse/project/square/fife_logo_square.png'
-        );
-    });
+// const collectionWithBrowse = require('./C179003030-ORNL_DAAC.json');
+// const { granuleWithBrowse } = JSON.parse(fs.readFileSync(`${__dirname}/C1711961296-LPCLOUD_granules.json`));
```

Comment:
There's a lot of commented out code that could be removed.

Reply:
Yea I'm going to remove everything after I'm satisfied with my SIT testing